multi/imxrt-multi/spi.c

```diff
@@ -395,9 +395,18 @@ static int spi_getIsel(int mux, int *isel, int *val)
 static int spi_muxVal(int spi, int mux)
 {
 	if ((mux >= pctl_mux_gpio_lpsr_06 &&
-			mux <= pctl_mux_gpio_lpsr_08) ||
-			mux == pctl_mux_gpio_lpsr_12)
-		return (spi == 5) ? 4 : 8;
+					mux <= pctl_mux_gpio_lpsr_08) ||
+					mux == pctl_mux_gpio_lpsr_12) {
+		if (spi == 5) {
+			return 8;
```

Comment:
Why return 8 for `spi == 5`? The previous expression was `(spi == 5) ? 4 : 8`

Reply:
i.MX RT1170 Processor Reference Manual, Rev. 1, 05/2021, page 397/398. 
For LPSPI5 -> ALT8 or ALT1
For LPSPI6 -> ALT4
I see a shift in the spi numbering
`#if SPI1
		{ 0, PIN2MUX(SPI1_SCK) },`